.github/workflows/integration_test_ma_baseline_ddp.yml

```diff
@@ -0,0 +1,33 @@
+name: Integration Test Model Adapter baseline DDP
+
+on:
+  pull_request:
+    branches:
+    - main
+    paths:
+    - '.github/workflows/integration_test_ma_baseline_ddp.yml'
+    - 'conf/demo/baseline/**'
+    - 'Dockerfile-ubuntu18.04/DockerfilePytorch112'
+    - 'e2eAIOK/ModelAdapter/**'
+    - 'e2eAIOK/common/**'
+    - 'tests/cicd/ModelAdapterJenkinsfile'
+    - 'tests/cicd/jenkins_ma_baseline_ddp_test.sh'
+
+jobs:
+  integration_test:
+    name: Integration Test Model Adapter baseline DDP
+    runs-on: self-hosted
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: Build Docker
+      run: |
+        cd Dockerfile-ubuntu18.04 && docker build -t e2eaiok-pytorch112 . -f DockerfilePytorch112 && cd .. && yes | docker container prune && yes | docker image prune
+
+    - name : Test Model Adapter Baseline DDP
+      run: |
+        if [[ $(docker ps -q --filter "name=test-ma-baseline-ddp" | wc -l) -ne 0 ]]; then docker ps -q --filter "name=test-ma-baseline-ddp" | grep -q . && docker stop $(docker ps -a -q) && docker rm $(docker ps -a -q); fi
+        docker run --rm --name test-ma-baseline-ddp --shm-size=10g --privileged --network host --device=/dev/dri -v /mnt/DP_disk1/dataset/MA:/home/vmagent/app/data  -v `pwd`:/home/vmagent/app/e2eaiok -w /home/vmagent/app/ e2eaiok-pytorch112 /bin/bash -c ". /home/vmagent/app/e2eaiok/tests/cicd/jenkins_ma_baseline_ddp_test.sh"
```

Comment:
Did you hard-coded the # of nodes in ddp_test.sh? will it be a problem if the testing env is smaller?

Reply:
in ddp_test.sh, we start two processes on one node for testing, thus no worry for env change.